root.tf

```diff
@@ -261,8 +261,21 @@ module "ecr_consignment_api_repository" {
 
 module "ecr_transfer_frontend_repository" {
   source           = "./tdr-terraform-modules/ecr"
-  name             = "transfer-frontend"
-  image_source_url = "https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/Dockerfile"
+  name             = "aws-otel-collector"
+  image_source_url = "https://github.com/nationalarchives/tdr-xray-logging/blob/main/Dockerfile"
+  policy_name      = "transfer_frontend_policy"
+  policy_variables = {
+    intg_account    = data.aws_ssm_parameter.intg_account_number.value,
+    staging_account = data.aws_ssm_parameter.staging_account_number.value,
+    prod_account    = data.aws_ssm_parameter.prod_account_number.value
+  }
+  common_tags = local.common_tags
+}
+
+module "ecr_collector_repository" {
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "transfer-frontend-collector"
+  image_source_url = "https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/Dockerfile-collector"
   policy_name      = "transfer_frontend_policy"
```

Comment:
Will probably have to create a new policy with the correct permissions, but this one will do for now, until we try all this out.